Makefile

```
@@ -66,6 +70,15 @@ dev: ## Build and install a development build
 	@cp $(GOPATH)/bin/packer bin/packer
 	@cp $(GOPATH)/bin/packer pkg/$(GOOS)_$(GOARCH)
 
+lint: ## Lint Go code
```

```suggestion
lint: install-lint-deps ## Lint Go code
```
This one was missing it's dep